optimize/backend/src/main/java/io/camunda/optimize/license/CamundaLicense.java

```diff
@@ -40,6 +44,14 @@ public synchronized LicenseType getLicenseType() {
     return licenseType;
   }
 
+  public synchronized boolean isCommercial() {
+    return isCommercial;
+  }
+
+  public synchronized OffsetDateTime expiresAt() {
+    return validUntil;
+  }
```

Comment:
is there a reason we mix "valid until" and "expires at" rather than just using one term?

Reply:
`validUntil` is the existing term for expiration from the license checker, `expiresAt` is what the C3 Header expects and therefore what the API & backend uses

I'll update the variable to `expiresAt`